index.html

```diff
@@ -53,6 +53,230 @@ <h1>I'm Anold. <br>Glad to see <br> you!</h1>
+            </li>
+          </ul>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button type="button">See Project</button>
+      </article>
+      <article class="work">
+        <img src="images/Snapshoot Portfolio3.png" alt="">
+        <div>
+          <h2>Tonic</h2>
+        </div>
+        <div class="canopy">
+          <div>
+            <h5>CANOPY</h5>
+          </div>
+          <ul>
+            <li>
+              <img src="images/Counter.png" alt="counter image">
+              <span>Backend dev</span>
+            </li>
+            <li>
+              <img src="images/Counter.png" alt="counter image">
+              <span>2015</span>
+            </li>
+          </ul>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button type="button">See Project</button>
+      </article>
+      <article class="work">
+        <img src="images/Snapshoot Portfolio2.png" alt="">
+        <h2>Tonic</h2>
+        <div class="canopy">
+          <div>
+            <h5>CANOPY</h5>
+          </div>
+          <ul>
+            <li>
+              <img src="images/Counter.png" alt="counter image">
+              <span>Backend dev</span>
+            </li>
+            <li>
+              <img src="images/Counter.png" alt="counter image">
+              <span>2015</span>
+            </li>
+          </ul>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button type="button">See Project</button>
+      </article>
+      <article class="work">
+        <img src="images/Snapshoot Portfolio.png" alt="">
+        <h2>Tonic</h2>
+        <div class="canopy">
+          <div>
+            <h5>CANOPY</h5>
+          </div>
+          <ul>
+            <li>
+              <img src="images/Counter.png" alt="counter image">
+              <span>Backend dev</span>
+            </li>
+            <li>
+              <img src="images/Counter.png" alt="counter image">
+              <span>2015</span>
+            </li>
+          </ul>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button>See Project</button>
+      </article>
+    </section>
+
+    <section class="about_me">
+      <article class="about_myself">
+        <h1>About Myself</h1>
```

Comment:
Please note that the `h1`tag is used for the main header of your page and shouldn't be used more than once for a good semantics and search engine optimization, so kindly use a lower `hx` tag obeying the hierarchy principle for `header`tags 😄